server.py

```diff
@@ -0,0 +1,87 @@
+"""
+Main server script
+
+Has Database connection setup, config loading, and routes
+
+Author: Kyle Pekosh
+Copyright 2019 by Kyle Pekosh
+"""
+import json
+import logging
+import os
+import sys
+
+from flask import Flask
+
+import thorcast.thorcast as thorcast
+import thorcast.geocodex as gx
+import thorcast.weather_cache as wc
+
+
+GC_USERNAME = os.getenv('THORCAST_DB_USERNAME')
+GC_PASSWORD = os.getenv('THORCAST_DB_PASSWORD')
+GC_HOST = os.getenv('THORCAST_DB_HOST')
+GC_PORT = os.getenv('THORCAST_DB_PORT')
+GC_DB = os.getenv('THORCAST_DB_NAME')
+
+geocodex = gx.Geocodex(
+    GC_USERNAME,
+    GC_PASSWORD,
+    GC_HOST,
+    GC_PORT,
+    GC_DB
+)
+
+REDIS_HOST = os.getenv('REDIS_HOST')
+REDIS_PORT = os.getenv('REDIS_PORT')
+REDIS_DB = os.getenv('REDIS_DB')
+REDIS_PASSWORD = os.getenv('REDIS_PASSWORD')
+
+weather_cache = wc.WeatherCache(
```

Comment:
Most Redis clients support `REDIS_URL` which would reduce the number of ENV vars you'd need to manage.

Reply:
I'm not seeing that functionality in the Redis' official Python package. I'm not concerned about the env list. It's only a few more, and the only one likely to change is the host.